sdk/include/game-contract-sdk/game_base.hpp

```diff
@@ -368,6 +356,29 @@ class game : public eosio::contract {
         new_game_internal(ses_id, casino_id);
     }
 
+    CONTRACT_ACTION(newgamebon)
```

Comment:
do we need newgamebon without affiliates? we can send empty affiliate to newgamebonaf

Reply:
i'm not sure tbh

seems like we did this for `newgame` to support backward compatability

Reply:
fixed